gradle-baseline-java/src/main/groovy/com/palantir/baseline/extensions/BaselineErrorProneExtension.java

```diff
@@ -24,12 +24,13 @@
 
     private static final ImmutableList<String> DEFAULT_PATCH_CHECKS = ImmutableList.of(
             // Baseline checks
+            "LambdaMethodReference",
```

Comment:
If these are applied in order, should `LambdaMethoReference` come after `OptionalOrElseMethodInvocation`

Reply:
order doesn't appear to make a difference, either way it takes two runs to migrate to orElseGet + method reference.

Reply:
ok, good to know!